modules/tensor_mechanics/test/tests/action/tests

```diff
@@ -33,25 +33,23 @@
                   'subblocks requesting different strain formulations.'
   []
 
-  [error_unrestricted]
+  [two_block.error_unrestricted]
```

Comment:
We should _really_ be consistent about naming test blocks with the same names as our test inputs. Anything else is just confusing when looking for the inputs.